nixos/modules/misc/documentation.nix

```diff
@@ -187,7 +197,30 @@ in
       environment.systemPackages = [ pkgs.man-db ];
       environment.pathsToLink = [ "/share/man" ];
       environment.extraOutputsToInstall = [ "man" ] ++ optional cfg.dev.enable "devman";
-      environment.etc."man.conf".source = "${pkgs.man-db}/etc/man_db.conf";
+      environment.etc."man_db.conf".text =
+        let
+          manualPages = pkgs.symlinkJoin {
```

Comment:
Maybe `buildEnv` would be more suitable, since it allows linking only `/share/man` of each package and requires fewer symlinks, e.g.

```
buildEnv {
  name = "man-paths";
  paths = config.environment.systemPackages;
  pathsToLink = ["/share/man"];
  extraOutputsToInstall = ["man"];
  ignoreCollisions = true;
}
```

Reply:
Done, but I haven't tested it yet because some major package appears to be broken in staging.